### DISCUSSION TITLE
Tweaks gtk translations

### DISCUSSION THREAD

**Discussion by davidphilipbarr** [labwc/labwc#2369] [General]
i guess this is mostly for @01micko - I'm adding some new options to tweaks i guess mostly behaviour tweaks and (freedesktop) colour scheme stuff as well as some mouse stuff - do i just adds the _() to the titles and update the .po files?

images for fun,  i'm just rebasing it now so it's a bit old:

![20241120_18h57m06s_grim](https://github.com/user-attachments/assets/7c34ec5c-7092-476e-852b-4e579ab0c2d8)
![20241120_19h19m41s_grim](https://github.com/user-attachments/assets/d896657b-fc3e-4208-af81-45574f000c81)
![20241120_19h19m25s_grim](https://github.com/user-attachments/assets/11292089-6b20-4b25-9343-8f878c84701d)

still no fonts...

**Comment by 01micko** [comment]
Yes, pretty much.

| Old                    | New
|----------------------|-----------------------
| `("Some title")`  | `(_("Some title"))`
| `"Name"`           | `_("Name")`


Then when done run from the main dir:

```
xgettext --keyword=_  --add-comments -o po/labwc-tweaks-gtk.pot main.c \
stack-appearance.c stack-lang.c stack-mouse.c data/labwc-tweaks-gtk.desktop.in  \
--package-name=labwc-tweaks-gtk --package-version=0.1.0 \
--msgid-bugs-address=https://github.com/labwc/labwc-tweaks-gtk/issues
```

**Reply by 01micko** [reply]
Oh, if you add a new `stack-whatever.c` add it to the `xgettext` command above.

**Comment by davidphilipbarr** [comment]
I'd completely overlooked this, cheers. 

this is the branch I'm working on if you wanted to peek.

https://github.com/davidphilipbarr/labwc-tweaks-gtk/tree/more-options

**Reply by 01micko** [reply]
Had a look, one thing you forgot:

[NLS.md](https://github.com/labwc/labwc-tweaks-gtk/blob/master/NLS.md) (and I should fix that to add the file to the `xgettext` command)


3. After adding and testing your code additions to satisfaction, backup
`po/labwc-tweaks-gtk.pot`. You need the custom header from that file for
the newly generated .pot file in the next step.

...

5. Copy the header from the original `labwc.pot` to the new one, keeping
the newly generated dates, check for sanity and commit.

**Reply by 01micko** [reply]
https://github.com/labwc/labwc-tweaks-gtk/pull/31 addresses a few things to hopefully make things a bit easier.
